src/views/ScenarioManager/labels.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export const getScenarioManagerLabels = (t) => ({
+  status: t('commoncomponents.scenariomanager.treelist.node.status.label'),
+  successful: t('commoncomponents.scenariomanager.treelist.node.status.successful'),
+  running: t('commoncomponents.scenariomanager.treelist.node.status.running'),
+  failed: t('commoncomponents.scenariomanager.treelist.node.status.failed'),
+  created: t('commoncomponents.scenariomanager.treelist.node.status.created'),
+  delete: t('commoncomponents.scenariomanager.treelist.node.action.delete'),
+  redirect: t('commoncomponents.scenariomanager.treelist.node.action.redirect'),
+  scenarioRename: {
+    title: t('commoncomponents.dialog.create.scenario.input.scenarioname.label'),
+    errors: {
+      emptyScenarioName: t('commoncomponents.dialog.create.scenario.input.scenarioname.error.empty'),
+      forbiddenCharsInScenarioName: t(
+        'commoncomponents.dialog.create.scenario.input.scenarioname.error.forbiddenchars'
+      ),
+      existingScenarioName: t('commoncomponents.dialog.create.scenario.input.scenarioname.error.existing'),
+    },
+  },
+  deleteDialog: {
+    description: t(
+      'commoncomponents.dialog.confirm.delete.description',
+      'The scenario will be deleted. If this scenario has children, ' +
+        'then its parent will become the new parent of all these scenarios.'
```

Comment:
just a question: why some labels have a fallback value et others don't ?

Reply:
I don't know. Personally, I put it one systematically  but we should ask the question to THU.

Reply:
It's a good practice to set fallback values (in case the key is missing in some translation files), feel free to add them where they are missing :)